docs/reference/tags/form_tags/clear_cart/index.md

```diff
@@ -21,7 +21,7 @@ The `clear_cart` tag renders a button that clears the customer's cart.
 ##### output
 {% raw %}
 ```html
-<form action="/sites/cart" accept-charset="UTF-8" method="post">
+<form action="/sites/cart" date-action="clear_cart" accept-charset="UTF-8" method="post">
```

Comment:
@eballantine should this be `data-action` rather than `date-action`?

Reply:
Thanks @samantha-lind will fix

Reply:
@eballantine we are going to use `data-form-action` instead, just to ensure it doesn't clash with other js that we currently use that format.